src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -2,57 +2,66 @@
 
 import lotto.exception.LottoTicketException;
 
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class LottoTicket {
     private static final int COUNT_FOR_SECOND_RANK = 5;
     private static final int LOTTO_SIZE = 6;
+    private static final String DELIMITER = ",";
+
     private final List<LottoNumber> numbers;
 
-    public LottoTicket(List<LottoNumber> numbers) {
+    private LottoTicket(List<LottoNumber> numbers) {
+        this.numbers = numbers;
+    }
+
+    public static LottoTicket fromSixNumbers(List<LottoNumber> numbers) {
         validate(numbers);
         Collections.sort(numbers);
-        this.numbers = numbers;
+        return new LottoTicket(numbers);
+    }
+
+    public static LottoTicket fromInput(String input) {
+        List<LottoNumber> numbers = generateSixNumbersFromInput(input);
+        return fromSixNumbers(numbers);
     }
```

Comment:
```suggestion
     public static LottoTicket fromInput(String input) {
        List<LottoNumber> numbers = Arrays.stream(input.split(DELIMITER))
                .map(String::trim)
                .map(Integer::parseInt)
                .map(LottoNumber::new)
                .collect(Collectors.toList());
        return fromSixNumbers(numbers);
    }

```

`generateSixNumbersFromInput`로 빼지 않아도 충분할 것 같습니다. 스트림만으로도 어떤 연산을 통해 `numbers`를 만드는지 알수 있기때문에 메서드를 과도하게 추출한 경우로 보입니다.
메서드로 뺄지 말지 이런 부분이 어려울 수 있는 부분인데 단순하게 생각해서 메서드 추출을 하지 않아도 이해하는데 어렵지 않고, 다른 곳에서도 중복되어 사용되지 않는다면 일단 빼지 않는 것도 좋은 방법입니다. (보통은 거꾸로의 상황이 메서드 추출을 해야할 냄새가 나는 경우라 하죠)

Reply:
과도하게 추출된 메서드들을 다시 합쳐서 가독성을 해치지 않는 범위 내에서만 분리하였습니다!

Reply:
마틴 파울러의 리팩토링이라는 책에 보면 메서드를 정리하는 방법 중 서로 상반된 두가지를 제시합니다.
메서드 추출, 메서드 내용 직접 삽입
이 책은 테크코스 내 해당 도서가 있을 것 같은데 챕터6의 이 내용은 한번 쯤 보시면 도움이 될 것 같네요. 특히나 지금처럼 추출/삽입을 직접 해보신 상태라면 생각해볼 점들이 많을 걸로 보이네요.

참고로 책의 그 외의 내용은 당장 고민해보지 않은 부분들이 많아 직접적으로 지금 도움이 되긴 힘들 것 같아 추천 드리고 싶진 않습니다.

Reply:
책이 마침 집에 있어서 볼 수 있습니다! 말씀해주신 챕터 읽어볼게요~! 😄 👍

Reply:
피드백 과정을 가지지 않고 마무리 한 이유는 요구에 의해 적용해보는 것보다는 과제를 진행해야 한다는 부담감 없이 이번 단계때 고민 해본 상황이랑 해당 내용을 보시고 생각하는 시간을 가지는게 더 많은 도움이 될 거라고 생각됐네요.

내용 자체는 보면 당연한 얘기고 단순해서 어렵지 않은데 언제 어떻게가 힘든 부분이라 내용에 너무 몰두 하기보다는 추출/삽입 상황에 대해서 고민 해보는 시간을 가지는 것만으로도 많은 도움이 될 거라고 생각합니다.